### DISCUSSION TITLE
How to execute command in REPL?

### DISCUSSION THREAD

**Discussion by leetschau** [johnkerl/miller#1616] [Q&A] [answered]
Is it possible to run a command (e.g.: `flatten`) or chain of commands (e.g.: `flatten then cut -f ...`) in REPL?
If it is, how to run it? I tried the following but none of them works:
```
cat $*
:cat $*
```

Thanks!

**Comment by johnkerl** [comment, marked as answer]
@leetschau sorry, no -- the REPL lets you read records, and execute DSL statements. Esssentially it's an interactive `put` -- an interactive environment for that verb only.
